docker-compose.yml

```
@@ -25,6 +25,6 @@ services:
       - GETH_METRICS_ADDR=0.0.0.0
       - GETH_STATE_SCHEME=path
```

```suggestion
      - GETH_STATE_SCHEME=path
```
this is now going to be set by default, we dont need it necessarily anymore